core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -901,4 +903,32 @@ static function (array $allowedProperties): array {
             ->end()
         ;
     }
+
+    private function addAltchaNode(): NodeDefinition
+    {
+        return (new TreeBuilder('altcha'))
+            ->getRootNode()
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->enumNode('algorithm')
+                    ->info('The algorithm used to generate the challenges. Select between "SHA-256", "SHA-384" or "SHA-512".')
+                    ->values(Algorithm::values())
+                    ->defaultValue(Algorithm::sha256->value)
+                ->end()
+                ->integerNode('range_min')
+                    ->info('A higher value increases the complexity/security but may significantly increase the computational load on client devices, potentially impacting the user experience.')
+                    ->defaultValue(10000)
+                ->end()
+                ->integerNode('range_max')
+                    ->info('A higher value increases the complexity/security but may significantly increase the computational load on client devices, potentially impacting the user experience.')
```

Comment:
We should add a `->max(1_000_000)` here as this is the default max number of the wiget: https://github.com/altcha-org/altcha/blob/8a7978eaf67ae8d32d2b3a0be2a30ef257fe063c/src/Altcha.svelte#L41 otherwise we would have to pass the correct `maxnumber` to the widget.

Reply:
We already do:

<img width="592" alt="" src="https://github.com/contao/contao/assets/1192057/622963d8-49f2-4b1c-9453-1bef1fc9199f">

Reply:
The `$altchaMaxNumber` is `1e7` currently though, instead of `1e6`.

Reply:
And `$this->altchaMaxNumber` does not seem to be used anywhere?

Reply:
No. It was configurable in the first version of the PR, that is probably why the property exists.

Reply:
Fixed in e275af6cfd858892e314002c09b0c24540767d7d.